demo/demo-app.jsx

```diff
@@ -10,12 +10,46 @@ const wrapperComponent = () => (
   <React.Fragment>
     <div className="toolbar-pf row">
       <Toolbar
+        count={1}
```

Comment:
Can I ask what does the count does and why is it required?

Reply:
In manageiq the toolbar needs count of selected items in the GTL on the page.

Anyway: making it optional, adding default value.